packages/react-components/react-nav-preview/stories/NavDrawer/NavDrawer.stories.tsx

```diff
@@ -8,32 +8,63 @@ import {
   NavSubItem,
   NavSubItemGroup,
 } from '@fluentui/react-nav-preview';
-import { DrawerBody } from '@fluentui/react-drawer';
-import { makeStyles, shorthands, tokens } from '@fluentui/react-components';
+import { DrawerBody, DrawerProps } from '@fluentui/react-drawer';
+import { Button, Label, Radio, RadioGroup, makeStyles, shorthands, tokens, useId } from '@fluentui/react-components';
 import { Folder20Filled, Folder20Regular, bundleIcon } from '@fluentui/react-icons';
+
 const useStyles = makeStyles({
   root: {
+    ...shorthands.border('2px', 'solid', '#ccc'),
     ...shorthands.overflow('hidden'),
+
     display: 'flex',
-    height: '480px', // arbitrary value, for demo purposes only
-    // arbitrary value, for dramatic effect only
-    // I know this is ugly and wrong, but styling is coming, I promise.
-    backgroundColor: tokens.colorBackgroundOverlay,
+    height: '480px',
+    backgroundColor: '#fff',
+  },
+
+  content: {
+    ...shorthands.flex(1),
+    ...shorthands.padding('16px'),
```

Comment:
Should this be a token? I think it's spacing Large:
```suggestion
    ...shorthands.padding(tokens.spacingVerticalL, tokens.spacingHorizontalL),
```

Reply:
eh. Brandon and I are operating under the assumption that spacing tokens will go away when we roll out the new token strategy in a few months. They never work quite as well as people hope and end up causing a LOT of off by 1 errors.